test/fixtures/invalid-stringified-schema.js

```diff
@@ -1 +1,4 @@
-module.exports = `{"111!invalid string ... issue":''}`;
```

Comment:
You won't believe, but this is actually a valid JSON per `jju`.

Reply:
parsed as

```
{
  "111!invalid string ... issue": ""
}
```

? that would be pretty okay, they're allowing single quotes, apparently. The first part is bordered with `""` and that's valid.

Reply:
Yes, that was parsed into what you have written. While that's okay, it was failing the tests, since they expect this fixture to represent an _absolute monster invalid JSON_.

Reply:
Ha ha